BlueshiftSampleApp/ios/Podfile

```diff
@@ -1,5 +1,6 @@
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+require_relative '../node_modules/blueshift-react-native/process_install.rb'
```

Comment:
`process_post_install.rb` or `post_install.rb` ?

Reply:
renamed to post_install